examples/direct_fidelity_estimation.py

```diff
@@ -276,6 +277,24 @@ def _estimate_pauli_traces_general(qubits: List[cirq.Qid],
     return pauli_traces
 
 
+def _estimate_std_devs_clifford(fidelity: float, N: int):
+    """
+    Estimates the standard deviation of the measurement for Clifford circuits.
+
+    Args:
+        fidelity: The measured fidelity
+        N: the number of measurements
+    Returns:
+        The standard deviation (estimated and a bound)
```

Comment:
Could you add a note on what the bound means (i.e. it is statistical bound for within one standard deviation?).

Reply:
Like this?